src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 0.3s ease-in-out;
```

Лучше явно указывать свойство которое анимируеш
```suggestion
    transition: transform 0.3s ease-in-out;
```